packages/pulp-deb/pulp-deb.spec

```diff
@@ -6,15 +6,19 @@
 
 %define inst_prefix pulp_deb
 
+%if 0%{?fedora} >= 33
+%define __python /usr/bin/python2
+%endif
+
 
 # ---- archive related macros ----
 
-%define git_tag 1.10-release
+%define git_tag 1.10.2
```

Comment:
these should have been `%{version}`, but given this is what is built, I'll leave it as is.